airbyte_cdk/test/standard_tests/docker_base.py

```
@@ -0,0 +1,378 @@
+                tag=tag,
+                no_verify=False,
+            )
+
+        container_config_path = "/secrets/config.json"
+        container_catalog_path = "/secrets/catalog.json"
+
+        with (
+            scenario.with_temp_config_file(
+                connector_root=connector_root,
+            ) as temp_config_file,
+            tempfile.TemporaryDirectory(delete=False) as temp_dir_str,
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'delete' in constructor call

(E1123)